docs/source/getting-started/tabular-data/agg-and-filter.ipynb

```diff
@@ -0,0 +1,481 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(10, 6))\n",
+    "sns.lineplot(x=count_year_actual[\"YEAR\"].to_list(), y=count_year_actual[\"Actual Count\"].to_list(), marker=\"o\", label=\"Actual\")\n",
+    "sns.lineplot(x=count_year_dp[\"YEAR\"].to_list(), y=count_year_dp[\"DP Count\"].to_list(), marker=\"o\", label=\"DP\")\n",
+    "plt.title('Actual vs Differentially Private Counts by Year')\n",
+    "plt.xlabel('Year')\n",
+    "plt.ylabel('Count')\n",
+    "plt.legend(title='Legend')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The differentially private computed values align well with the actual counts; the trend that both of the counts demonstrate is also notable. It demonstrates the number of respondents increased significantly from 2008 to 2010. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Multiple Variable *`group_by`* "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To demonstrate the `group_by` method with multiple variables, let's compute the total number of respondents per year and quarter. We'll need a margin for the combination of the variables being used for the grouping. \n",
+    "\n",
+    "In this case, the margin is defined by the `(\"YEAR\", \"QUARTER\",)` entry in the initial definition of `context`. "
```

Comment:
I think some prior information on what a margin is, and maybe a link to some documentation would help here.